packages/remotemetrics/events.go

```diff
@@ -56,6 +62,29 @@ type MessageFinalizedMetrics struct {
 	LikeEdgeCount      int       `json:"likeEdgeCount,omitempty" bson:"likeEdgeCount"`
 }
 
+// MessageScheduledMetrics defines the scheduling message confirmation metrics record that is sent to remote logger
+type MessageScheduledMetrics struct {
+	Type               string    `json:"type" bson:"type"`
+	NodeID             string    `json:"nodeID" bson:"nodeID"`
+	MetricsLevel       uint8     `json:"metricsLevel" bson:"metricsLevel"`
+	MessageID          string    `json:"messageID" bson:"messageID"`
+	IssuedTimestamp    time.Time `json:"issuedTimestamp" bson:"issuedTimestamp"`
+	SolidTimestamp     time.Time `json:"solidTimestamp,omitempty" bson:"solidTimestamp"`
+	ScheduledTimestamp time.Time `json:"scheduledTimestamp,omitempty" bson:"scheduledTimestamp"`
+	BookedTimestamp    time.Time `json:"bookedTimestamp" bson:"bookedTimestamp"`
+	QueuedTimestamp    time.Time `json:"queuedTimestamp" bson:"queuedTimestamp"`
+	DroppedTimestamp   time.Time `json:"droppedTimestamp,omitempty" bson:"DroppedTimestamp"`
```

Comment:
Add delta fields as well, whose value will be set to difference between each timestamp and issued timestamp. 
Also add received timestamp.

Add delta that indicates processing time (scheduling time - received time)
Add delta that indicates time spent in the scheduler (scheduling time - queued timestamp)

Reply:
done

Reply:
Also add DeltaSolid DeltaScheduled DeltaBooked DeltaConfirmed as a difference between each event and issue timestamp

Reply:
Also add StrongEdgeCount  WeakEdgeCount LikeEdgeCount TransactionID so that it contains the same fields as MessageFinalizedMetrics. Or maybe create a shared structure that will be sent to two different indices (after scheduling and after confirmation), since ideally we want the same data for both of them.

Reply:
sounds weird to me that we should be sending the same data twice
especially since anything that was finalized neccesarily was either scheduled or dropped.

So I am keeping it seperate for now